src/api/app/models/kiwi/image.rb

```diff
@@ -157,6 +158,15 @@ def self.binaries_available(project, use_project_repositories, repositories)
 
   private
 
+  def kiwi_indentation(xml)
```

Comment:
Not sure if I really like having this in the model here. It looks like view logic to me.. I think we should start to think about seperating the business logic about kiwi images from the view logic about xml output. At the very least we could extract the xml presentation stuff into its own class like `Kiwi::Image::ToXml` or `Kiwi::Image::XmlFormatter` etc.